plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -20,6 +20,10 @@ def client
       deploy_group.kubernetes_cluster.client
     end
 
+    def job?
+      deploy_yaml.resource_name == 'job'
```

Comment:
doh ... yeah :D

On Sun, Jun 26, 2016 at 8:38 PM, rata <notifications@github.com> wrote:

> In plugins/kubernetes/app/models/kubernetes/release_doc.rb
> <https://github.com/zendesk/samson/pull/1092#discussion_r68519381>:
>
> > @@ -20,6 +20,10 @@ def client
> >        deploy_group.kubernetes_cluster.client
> >      end
> >
> > +    def job?
> > +      deploy_yaml.resource_name == 'job'
>
> resource_name? Maybe it's better (in a future patch) to use
> "resource_kind" to avoid confusions with the "name" metadata of the object?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1092/files/9969624bc0f4beda19189ff258f9403db3bbfa71#r68519381>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ-O0BFfh9sv1NHA2Jzt8O9GMPnEyks5qP0XBgaJpZM4I8WmU>
> .
>